index.js

```diff
@@ -1,16 +1,13 @@
 module.exports = {
   presets: [
-    require('modify-babel-preset')('babel-preset-es2015', {
-      'transform-es2015-classes': { loose: true },
-      'transform-es2015-modules-commonjs': { loose: true }
-    }),
+    require('babel-preset-es2015'),
     require('babel-preset-react'),
     require('babel-preset-stage-0')
   ],
   plugins: [
     require('babel-plugin-transform-decorators-legacy').default,
     require('babel-plugin-add-module-exports'),
-    [require('babel-plugin-transform-es2015-template-literals'), { spec: true }],
+    [ require('babel-plugin-transform-es2015-template-literals'), { spec: true } ],
```

Comment:
Just a suggestion - es2015 already brings this in.  `["es2015", { "spec": true }]` should do the trick.